static/app/views/alerts/wizard/index.tsx

```diff
@@ -180,19 +180,21 @@ class AlertWizard extends Component<Props, State> {
             <WizardBody>
               <WizardOptions>
                 <Styledh2>{t('Errors')}</Styledh2>
-                {AlertWizardOptions.map(({categoryHeading, options}, i) => (
-                  <OptionsWrapper key={categoryHeading}>
-                    {i > 0 && <Styledh2>{categoryHeading}</Styledh2>}
-                    <RadioPanelGroup
-                      choices={options.map(alertType => {
-                        return [alertType, AlertWizardAlertNames[alertType]];
-                      })}
-                      onChange={this.handleChangeAlertOption}
-                      value={alertOption}
-                      label="alert-option"
-                    />
-                  </OptionsWrapper>
-                ))}
+                {alertWizardCategories(organization).map(
```

Comment:
What about calling this `getAlertWizardCategories`?

Reply:
sounds good, let me change that 👍